rubygem-foreman_bootdisk/rubygem-foreman_bootdisk.spec

```diff
@@ -82,16 +83,20 @@ cat <<GEMFILE > %{buildroot}%{foreman_bundlerd_dir}/%{gem_name}.rb
 gem '%{gem_name}'
 GEMFILE
 
+%foreman_precompile_plugin -a
```

Comment:
This is failing, as the rake task's trying to initialise the database, and we don't have foreman-sqlite as a build dep (thankfully?).

<pre>
+ /usr/bin/ruby193-rake 'plugin:apipie:cache[foreman_bootdisk]' RAILS_ENV=development cache_part=resources OUT=/builddir/build/BUILDROOT/ruby193-rubygem-foreman_bootdisk-4.0.2-1.git.188.6c277bf.el7.centos.x86_64/opt/rh/ruby193/root/usr/share/gems/gems/foreman_bootdisk    -4.0.2/public/apipie-cache/plugin/foreman_bootdisk
Rails Error: Unable to access log file. Please ensure that /builddir/build/BUILD/rubygem-foreman_bootdisk-4.0.2/usr/share/foreman/log/development.log exists and is chmod 0666. The log level has been raised to WARN and the output directed to STDERR until the problem is     fixed.
rake aborted!
Please install the sqlite3 adapter: `gem install activerecord-sqlite3-adapter` (Could not find sqlite3 (~> 1.3.5) amongst [actionmailer-3.2.8, actionpack-3.2.8, activemodel-3.2.8, activerecord-3.2.8, activeresource-3.2.8, activesupport-3.2.8, ancestry-2.0.0, apipie-ra    ils-0.2.6, arel-3.0.2, audited-3.0.0, audited-activerecord-3.0.0, bigdecimal-1.1.0, builder-3.0.0, bundler-1.3.5, bundler_ext-0.3.0, coffee-rails-3.2.2, coffee-script-2.2.0, coffee-script-source-1.3.3, deep_cloneable-2.0.2, diff-lcs-1.1.3, erubis-2.7.0, execjs-1.4.0,     fast_gettext-0.9.2, foreigner-1.7.1, foreman_bootdisk-4.0.2, friendly_id-4.0.10.1, gettext_i18n_rails-1.2.0, hike-1.2.1, i18n-0.6.0, io-console-0.3, journey-1.0.4, json-1.5.5, ldap_fluff-0.3.3, mail-2.4.4, mime-types-1.19, minitest-2.5.1, multi_json-1.10.1, net-http-p    ersistent-2.7, net-ldap-0.10.0, netrc-0.7.7, oauth-0.4.7, polyglot-0.3.3, rabl-0.11.4, rack-1.4.1, rack-cache-1.2, rack-ssl-1.3.2, rack-test-0.6.1, rails-3.2.8, railties-3.2.8, rake-0.9.2.2, rdoc-3.9.5, rest-client-1.6.7, ruby2ruby-2.1.3, ruby_parser-3.6.3, safemode-1    .2.2, scoped_search-2.7.1, secure_headers-1.4.1, sexp_processor-4.4.4, sprockets-2.8.2, thor-0.18.1, tilt-1.3.3, treetop-1.4.10, turbolinks-2.5.3, tzinfo-0.3.33, validates_lengths_from_database-0.4.0, will_paginate-3.0.2])
Tasks: TOP => plugin:apipie:cache
(See full trace by running task with --trace)
</pre>

Is there any way we can fix the rake task not to initialise the database connection, or is it necessary?